scripts/automation/tests/brief/specialist.js

```diff
@@ -0,0 +1,30 @@
+const buyerLogin = require('../../useCases/login/buyer');
+const specialistBrief = require('../../useCases/brief/specialist');
+const buyerDashboard = require('../../useCases/dashboard/buyer');
+
+describe('should be able to create specialist brief', function () {
```

Comment:
general comment - try to ES6-ify this code eg 
- `function` should be `() =>`
- `exports.create` should be `export const create`

Reply:
fixed